az-digital/az_quickstart
# Closes #943: Adding Google Tag Manager

This PR adds the google tag manager contrib module and creates a container that includes the GTM ID for the Arizona Comprehensive property.

## Description
In Quickstart 1, we had had the module for Google Tag manager installed by default along with configuration for the tag manager container to point to the Arizona Comprehensive GTM.

This PR recreates this functionality and will allow analytics tracking by default for all Quickstart 2 sites (assuming site owners don't disable the module or remove/edit the container).

## Related Issue
#943

## How Has This Been Tested?
Tested locally, in Probo, and in Google Analytics.

Probo site: https://7b5e2985-cedf-4a13-ab91-3a35916c835d--pr-955.probo.build/

To test:

1. Verify that tag manager is enabled and the Arizona Comprehensive container is present
2. Create a page
3. Verify that the GTM snippet appears in the page source
4. (if you have access to the Arizona Comprehensive analytics) Verify that traffic to pages on Probo site appear under realtime traffic (when not signed in)
5. Verify that traffic when signed in does not appear in GA.

![image](https://user-images.githubusercontent.com/10873398/126573849-ea7e68b8-4c27-427a-8d76-19d1a7eadbd7.png)


## Types of changes


## Checklist:
